compile/compile_test.go

```diff
@@ -781,6 +786,41 @@ func TestCompilerPlanTargetPruneUnused(t *testing.T) {
 	})
 }
 
+func TestCompilerPlanTargetUnmatchedEntrypoints(t *testing.T) {
+	files := map[string]string{
+		"test.rego": `package test
+
+		p := 7
+		q := p + 1`,
+	}
+
+	test.WithTempFS(files, func(root string) {
+
+		compiler := New().WithPaths(root).WithTarget("plan").WithEntrypoints("test/p", "test/q", "test/no")
```

Comment:
Do we already have "happy path" test cases? In this example, `test` is an entrypoint that should work, I suppose.

Reply:
It should, but I can add one where only the happy path is hit 👍

Reply:
There were plenty of happy path tests for this already, so left it out.